### DISCUSSION TITLE
Crazyflie 1

### DISCUSSION THREAD

**Discussion by dins1961** [bitcraze/discussions#1704] [Q&A]
I have a couple of Crazyflie 1's and I would like to get them flying again.
Not used them for a few years.
I have a crazyflie 2 that I use. 
Does anyone have any suggestions about what I need to do? 
Thanks .. Davy

**Comment by gemenerik** [comment]
Hi Davy,

It’s great to hear you still have some Crazyflie 1s! If you’re facing mechanical issues, we still offer some [spare parts for Crazyflie 1](https://store.bitcraze.io/collections/spare-parts-crazyflie-1-0), though unfortunately, spare motors are no longer available. You might need to throw in some new batteries, too. For software, support has ended, so you'll need to use older versions. STM32 firmware updates, for example, stopped with [2017.06](https://github.com/bitcraze/crazyflie-firmware/releases/tag/2017.06), as noted in [this issue](https://github.com/bitcraze/crazyflie-firmware/issues/249).

Happy tinkering
Rik

**Reply by dins1961** [reply]
Hello Rik
I'm aware of this "For software, support has ended, so you'll need to use older versions."
What I'd like to know is how to use the older version without compromising the current VM - can I run 2 VM's at the same time or is there another way of approaching this ?

**Reply by gemenerik** [reply]
You likely can have multiple versions of the VM in VirtualBox, but running them simultaneously might lead to issues. @ataffanel would be the best person to provide more insights on this once he’s back in the office.

Could you elaborate on your goals? Am I right in understanding that you’re aiming to fly both the Crazyflie 2 and the Crazyflie 1s simultaneously?
